src/web/module/history/context/DataContext.tsx

```diff
@@ -18,7 +19,7 @@ const DataProvider = ({ children }: { children: ReactNode }): JSX.Element => {
 
   function fetchData(): void {
     // @ts-expect-error DataProvider fetchData
-    Index().then((result: History[]) => setData(result));
+    index().then((result: History[]) => setData(result));
```

Comment:
no checking?
```suggestion
    index().then((result: History[]) => {
        if (result) setData(result);
    });
```

Reply:
yes, no checking for now, because added conditional if need to handle undefined value at other code, and since it's from indexed db it should be safe for now I think